### DISCUSSION TITLE
TP2 Ejercicio 7. Poner la potencia

### DISCUSSION THREAD

**Discussion by Elias-Turner** [INGCOM-UNRN-P1/cursada-2023#77] [Preguntas y respuestas] [answered]
chicos alguien sabe como poner un numero elevado a otro?

**Comment by ramiroLopezA** [comment]
No se para que lo utilizarias en este caso, el ejercicio (segun lo entendi yo) pide que realices una potencia pero sin utilizar el operador de *. No se si eso responde tu pregunta

**Reply by Elias-Turner** [reply]
Si lo se la cosa es que lo quiero usar como condición. es decir:

```C
while(numero<numero(simbolo de potencia) a tal numero){
    numero += numero
}
```

A eso me refiero solo lo necesito como parámetro no es que lo voy a usar para la potencia

**Reply by ramiroLopezA** [reply, marked as answer]
yo lo que hice es hacer una funcion que multiplique sin utilizar el * y despues para representar una potencia podes usar esa misma funcion y escribir la potencia como producto
